From fc8f0b5c9cb8277950a2fefdb7f754c47b172dfd Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Tue, 1 Nov 2011 22:29:48 +0100
Subject: [PATCH] units: drop [Install] section from remote-fs-pre.target

remote-fs-pre.target is not a unit a user should ever explicitly enable.
Instead services which need to hook before network mounts should pull it
in.
---
 units/remote-fs-pre.target |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/units/remote-fs-pre.target b/units/remote-fs-pre.target
index 5406aa2..8aceb08 100644
--- a/units/remote-fs-pre.target
+++ b/units/remote-fs-pre.target
@@ -10,6 +10,3 @@
 [Unit]
 Description=Remote File Systems (Pre)
 After=network.target
-
-[Install]
-WantedBy=multi-user.target
-- 
1.7.7

